recipes/epics-base/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "R3.14.12.4" %}
+package:
+    name: epics-base
+    version: {{ version }}
+
+source:
+    fn: base{{ version }}.tar.gz
+    url: http://www.aps.anl.gov/epics/download/base/base{{ version }}.tar.gz
+    sha256: 0ea4a74e83ff73d68b64de89bedf7c42102edc2cc69c69fd49eb35c04dc9e3de
+    patches:
+        - config_site.diff
+        - config_common.diff
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - readline
+  run:
+    - readline
+
+test:
+  commands:
+    - caget -h
+    - caput -h
+    - camonitor -h
+    - softIoc -h
+about:
+    home: http://www.aps.anl.gov/epics
+    license: Epics Open License
```

Comment:
Any chance we could change this behaviour @msarahan so that if the license file isn't found in the recipe dir, it looks in the source dir?